arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1208,18 +1236,18 @@
       }
     },
     "CloudToDeviceProperties": {
-      "description": "The Iot Hub Cloud-To-Device messaging properties.",
+      "description": "The IoT hub cloud-to-device messaging properties.",
       "type": "object",
       "properties": {
         "maxDeliveryCount": {
-          "description": "The max delivery count for the device queue. Range : 1-100.",
+          "description": "The max delivery count for cloud-to-device messages in the device queue. See: https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging#cloud-to-device-messages.",
```

Comment:
The doc in the url says that the default value is 100. Please update the swagger spec accordingly. This is a general comment for all such properties. Please do this everywhere in the spec.